src/main/java/com/xiaomi/infra/pegasus/client/HashKeyData.java

```diff
@@ -25,6 +26,11 @@ public HashKeyData(byte[] hashKey, List<Pair<byte[], byte[]>> values) {
     this.values = values;
   }
 
+  public HashKeyData(boolean allFetched, byte[] hashKey, List<Pair<byte[], byte[]>> values) {
+    this(hashKey, values);
```

Comment:
Is there any other place invoke the constructor above (`public HashKeyData(byte[] hashKey, List<Pair<byte[], byte[]>> values)`)?

Reply:
as you mentioned, it can be deleted